website/cue/reference/releases/0.20.0.cue

```diff
@@ -0,0 +1,649 @@
+							"name" : "my_db",
+							"operation" : "insert",
+						}
+					}
+				  ```
+				- The `date` filter now supports the `d` and `y` shorthands. For example:
+
+				  ```coffeescript
+				    parse_groks("Nov 16 2020 13:41:29 GMT", patterns: ["%{date("MMM d y HH:mm:ss z"):field}"] )
+				  ```
+
+				  Will yield:
+
+				  ```json
+				  1605534089000
+				  ```
+				- Numeric values are now returned as integers when there is no loss in precision; otherwise they
+				  continue to be returned as floats.
+				- Aliases that match multiple fields now correctly extract as an array of values.
+				  extracted values.
+				- Aliases with a filter now correctly extract.
+				- The `keyvalue` filter now correctly ignores keys without values.
+				- To match the beginning and the end of lines, `\\A` and `\\Z` must be used rather than `^` and $`
+				- The `keyvalue` filter now correctly handles parsing keys that start with a number
+				"""
+			pr_numbers: [10537, 10954, 10958, 10961, 10938, 10956, 10538]
+		},
+		{
+			type: "enhancement"
+			scopes: ["observability", "route transform", "remap transform", "datadog_agent source"]
+			description: """
+				The telemetry for a few components was updated as a result expanding the use of named outputs in Vector for components with multiple output streams:
+
+				- `route`
+				- `datadog_agent` (when `multiple_outputs` is `true`)
+				- `remap` (when `reroute_dropped` is `true`)
+
+				These components now add an `output` tag to their metrics. See [highlight article for named output
+				metrics](2022-01-19-component-sent-metrics-output-tag).
+				"""
+			pr_numbers: [10869]
+		},
+		{
+			type: "enhancement"
+			scopes: ["observability", "vrl"]
+			description: """
+				The `abort` function can now take an optional string message to include in logs and the metadata for
+				dropped events. For example:
+
+				```coffeescript
+				if .foo == 5 {
+					abort "foo is 5!"
+				}
+				```
+
+				The `foo is 5!` message will appear in Vector's internal logs as well as being used as
+				`metadata.message` for rerouted dropped events (if `reroute_dropped` is `true`).
+				"""
+			pr_numbers: [10997]
+		},
+		{
+			type: "enhancement"
+			scopes: ["observability", "remap transform"]
+			description: """
+				If a custom message is passed to `assert` or `assert_eq`, this message is now used as `metadata.message`
+				rather than the default VRL error message which included `function call error: ...`. This allows cleaner
+				handling of different assertion errors if routing dropped events to another component (when
+				`reroute_dropped` is `true`).
+				"""
+			pr_numbers: [10914]
+		},
+		{
+			type: "enhancement"
+			scopes: ["vector source", "vector sink"]
+			description: """
+				The `vector` source and `vector` sink now default to version 2 of the protocl for inter-Vector
+				communication. See the [upgrade guide](highlights/2022-02-08-0-20-0-upgrade-guide#deprecate-v1) for more
+				details.
```

Comment:
It is, but we typically haven't documented how to skip versions. When users are upgrading Vector it is assumed they are upgrading one point release at a time. Later this will be one major-version at a time when we hit 1.0 and point releases won't have backwards incompatible changes.